Procurement/ViewModel/Recipes/RecipeManager.cs

```diff
@@ -28,6 +28,7 @@ public RecipeManager()
                 new SameBaseTypeRecipe(),
                 new SameNameRecipe("Alchemy Orb - 3 Of The Same Name", 3),
                 new SameNameRecipe("Chance Orb - 2 Of The Same Name", 2),
+                new VaalOrbRecipe()
```

Comment:
Style issue, but I would recommend adding a trailing comma to this line, to minimize the number of changed lines when another recipe is added.

Reply:
Minor but I like to keep you happy xD